### PR TITLE
copr: Add vectorized left_shift

### PR DESCRIPTION
**Pull request by gauss1314**
PCP #5751 

###  What have you changed?

add vectorized left_shift

###  What is the type of the changes?

- Improvement (a change which is an improvement to an existing feature)

###  How is the PR tested?
- Unit test
- Integration test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No

###  Does this PR affect `tidb-ansible`?

No

### CODE REVIEW

File: components/tidb_query/src/rpn_expr/impl_op.rs [hunk 1]
```diff
@@ -422,4 +431,24 @@ mod tests {
             assert_eq!(output, expect_output, "{:?}, {:?}", arg, sig);
         }
     }
+
+    #[test]
+    fn test_left_shift() {
+        let cases = vec![
```
Comment (andylokandy) [inline components/tidb_query/src/rpn_expr/impl_op.rs]: Suggest to add `None, None => None`

**Comment by gauss1314** [comment]
@sre-bot /run-integration-copr-test

**Comment by lonng** [comment]
@gauss1314 You should be file an issue in github.com/tikv/copr-test and add the `left_shift` to the `https://github.com/tikv/copr-test/blob/master/push-down-test/functions.txt`

**Comment by gauss1314** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/31

**Pull request merged**